### DISCUSSION TITLE
Vercel 504 Gateway timeout  on fallback route

### DISCUSSION THREAD

**Discussion by BernardA** [vercel/next.js#25863] [Help]
### What version of Next.js are you using?

10.2.3

### What version of Node.js are you using?

14.15.4

### What browser are you using?

Chome, Firefox

### What operating system are you using?

macOS

### How are you deploying your application?

Vercel

### Describe the Bug

Getting 504 - Gateway timeout when attempting to access fallback route on Vercel. 

The pre-built routes work fine. 

This also works fine when running `build` and `start` on localhost. So the issue seems to be specific to Vercel. 

No build errors occur. 

This is an instance of the error:

<img width="1769" alt="Screenshot 2021-06-05 at 09 48 20" src="https://user-images.githubusercontent.com/9449665/120887456-42994d80-c5f3-11eb-8e7a-efcdcf0a4cfe.png">

### Expected Behavior

Should normally render fallback route.

### To Reproduce

Attempt to access 
https://soeezauto-client.vercel.app/fiches-techniques/audi/q8. 

That should throw the error. 

This route that was pre-built should work normally: 
https://soeezauto-client.vercel.app/fiches-techniques/bmw/serie-5

Full code at [this repository](https://soeezauto-client.vercel.app/fiches-techniques/bmw/serie-5)

The specific file is at: `/src/pages/fiches-techniques/[brand]/[model].js

**Comment by cloudorbush** [comment]
Experiencing same issue scrolling through a list of dynamically routed ```<Link/>``` components.

When scrolling through the list slowly, the prefetch successfully resolves to 200 status code.

But when I scroll fast through the list, which creates a backlog of multiple dynamic pages to be prefetched, those requests all come back with a 504 status code response.
![image](https://user-images.githubusercontent.com/57633368/120906102-aee07500-c624-11eb-8d8e-e91976e9ea00.png)

**Comment by timneutkens** [comment]
Hey! Could you create a support case here: https://vercel.com/support/request so that the customer success team can help with your app?

**Reply by dainius-zm** [reply]
Hi, has the issue been resolved as we are experiencing the same problem?
